app/controllers/movies_controller.rb

```diff
@@ -1,4 +1,17 @@
 class MoviesController < ApplicationController
+  def show
+    conn = Faraday.new(url: 'https://api.themoviedb.org') do |f|
+      f.params['api_key'] = ENV['MOVIE_API_KEY']
+    end
+    movie_id = params[:id]
+    movie_stats = conn.get("/3/movie/#{movie_id}")
+    @movie_stats = JSON.parse(movie_stats.body, symbolize_names: true)
+    movie_reviews = conn.get("/3/movie/#{movie_id}/reviews")
+    @movie_reviews = JSON.parse(movie_reviews.body, symbolize_names: true)
```

Comment:
These will eventually live in the MovieService that we create (after we do the refactoring lesson)